internal/cli/utils/test/test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	cmdUtils "github.com/perses/perses/internal/cli/utils"
+	"github.com/perses/perses/pkg/client/api"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v2"
+)
+
+func JSONMarshalStrict(obj interface{}) []byte {
+	if data, err := json.Marshal(obj); err != nil {
+		panic(err)
+	} else {
```

Comment:
Omit the else when possible after error handling, see https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow.